faq.html

```diff
@@ -132,6 +132,22 @@ <h2>What do I need to bring?</h2>
         </p>
       </section>
 
+      <section class="left">
+        <h2>Is there internet?</h2>
+        <p>
+          There is some WiFi at Mt Cheeseman, and people have taped their cellphones to 
+          a window in the past to get mobile data coverage but the organisers can't guarantee
+          connectivity, speed, or stability. If you want to code please bring your laptop 
+          loaded up with the software you need.
+        </p>
+        <p>
+          We are however very likely to have Starlink installed. This hasn't been tested as we don't have a current paid plan for it, and due to the venue currently being used as an active ski area (which you can and should support this Winter!). 
+        </p>
+        <p>
+          We will do our best to get to the venue a couple of weeks beforehand to install and test connectivity so that attendees know beforehand.
```

Comment:
```suggestion
          We will do our best to get to the venue a couple of weeks beforehand to install and test connectivity so that attendees know what to expect before the event.
```